fireeye_service/__init__.py

```diff
@@ -213,9 +213,10 @@ def get_analysis(self):
         first = True
         while counter <= 5:
             r = requests.get(self.base_url + '/submissions/status/' + self.task, headers=headers, verify=False, proxies=self.proxies)
+            res = r.json()['submissionStatus']
             if first is True:
-                time.sleep(self.timeout+20)
-            if r.text == "Done":
+                time.sleep(self.timeout+10)
+            if res.text == "Done":
```

Comment:
r.text in this case is either {"submissionStatus": "In Progress"} or {"submissionStatus": "Done"}
by using json, I'm just splicing out the status field